### PR TITLE
Dread: Revise logic for the Screw Attack area and some more East

### PR DESCRIPTION
**Pull request by XenoWars**
Screw Attack Area
- Added Speed and Dboost methods of climbing Freezer
- Added WBJ methods where appropriate
- Added the Water Space Jump to Start Point 2
- Added triggering the flipper from below with beginner Knowledge
- Added pseudo-wave to the underwater blob in both directions
- Corrected Door to Transport to Burenia dock type
- Added various methods of not triggering the flipper
- Added event node to pull the grapple block from behind to the Cataris Teleport

East Artaria
- Added Ice Missile method to cross Lower Path to Cataris
- Added Speed method to go up Transport to Cataris
- Added Speed method for crossing Lower Path to Cataris by stopping the Cloak door from closing

Other
- Changed Energy Recharge Station South single-wall wall jumps to beginner

### CODE REVIEW

File: randovania/games/dread/json_data/Artaria.txt [hunk 1]
```diff
@@ -5062,12 +5068,12 @@ Extra - asset_id: collision_camera_081
   * Extra - right_shield_def: None
   > Tile Group (SCREWATTACK) 2
       After Artaria - Screw Attack Rotatable
-  > Tile Group (WEIGHT) 3
-      Morph Ball and After Artaria - Screw Attack Rotatable
```
Comment (kirbymastah) [inline randovania/games/dread/json_data/Artaria.txt:5066]: Reason behind removing this? I thought you should be able to reach these crumble blocks with morph ball but no screw attack?

File: randovania/games/dread/json_data/Artaria.txt [hunk 2]
```diff
@@ -5344,15 +5390,18 @@ Extra - asset_id: collision_camera_082
           Any of the following:
               Gravity Suit
               Heat/Cold Runs (Intermediate) and Cold Damage ≥ 50
-          Walljump (Beginner) or Simple IBJ or Use Spin Boost
+          Any of the following:
+              Walljump (Beginner) or Simple IBJ or Use Spin Boost
+              Flash Shift and Speed Booster
```
Comment (kirbymastah) [inline randovania/games/dread/json_data/Artaria.txt:5395]: Where would you charge the speedbooster from this door? Does it require frozen artaria?

**Review by kirbymastah — CHANGES_REQUESTED** [review]
Few questions

File: randovania/games/dread/json_data/Artaria.txt (hunk 1 above)
Comment (XenoWars) [reply randovania/games/dread/json_data/Artaria.txt:5066]: The door goes to SCREWATTACK 2 and SCREWATTACK 2 goes to WEIGHT 3, therefore this connection is now redundant

Comment (XenoWars) [reply randovania/games/dread/json_data/Artaria.txt:5066]: unless im misunderstanding things

File: randovania/games/dread/json_data/Artaria.txt (hunk 2 above)
Comment (XenoWars) [reply randovania/games/dread/json_data/Artaria.txt:5395]: You charge it at the bottom of the room/Red Chozo Arena. Jump up the first ledge, jump up against the wall, and spark diagonally then flash to ledge. But seeing as how beginner walljumps (with no other items for assistance) are being treated as trivial now, that will no longer matter. I will make this and all other walljumps i've done into trivial after the pr is in

**Review by kirbymastah — APPROVED** [review]

**Pull request merged**